homeassistant/helpers/update_coordinator.py

```diff
@@ -285,8 +286,12 @@ async def async_config_entry_first_refresh(self) -> None:
         to ensure that multiple retries do not cause log spam.
         """
         if self.config_entry is None:
-            raise ValueError(
-                "This method is only supported for coordinators with a config entry"
+            report(
+                "uses `async_config_entry_first_refresh` without a config entry, "
```

Comment:
Can this be rephrased so it's clearer the problem it's the coordinator's config entry which is missing? It was clearer in the exception text IMO.

Reply:
> Can this be rephrased so it's clearer the problem it's the coordinator's config entry which is missing? It was clearer in the exception text IMO.

I've updated the error message.